rpmbuild/copr-rpmbuild.spec

```diff
@@ -1,4 +1,4 @@
-%if 0%{?fedora} || 0%{?rhel} > 7
+%if 0%{?fedora} || 0%{?rhel} > 7 || 0%{?openEuler}
```

Comment:
I'd drop this as well;  let's drop Python2 support with EL7 support.

Reply:
> I'd drop this as well; let's drop Python2 support with EL7 support.

Done